sessions/js-variables-and-numbers/simple-calculator/js/index.js

```diff
@@ -0,0 +1,74 @@
+console.clear();
+
+const operand1 = 12;
+const operand2 = 4;
+
+/*
+Mathematical operations
+*/
+const addButton = document.querySelector('[data-js="add"]');
+const subtractButton = document.querySelector('[data-js="subtract"]');
+const multiplyButton = document.querySelector('[data-js="multiply"]');
+const divideButton = document.querySelector('[data-js="divide"]');
+const exponentButton = document.querySelector('[data-js="exponent"]');
+const moduloButton = document.querySelector('[data-js="modulo"]');
+
+addButton.addEventListener('click', () => {
+	// 1. Add the two operands and store the result in a variable. Log variable's value to the console.
```

Comment:
In each instance, it'd be "log the variable's[...]" :)

Reply:
Good catch!